exampleExercises/src/Date/DateDemo.java

```diff
@@ -0,0 +1,27 @@
+package Date;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class DateDemo {
+    public static void main(String[] args) {
+        Calendar calendar = Calendar.getInstance();
```

Comment:
@maria-salmasian is that a other project?

Reply:
yes, idk why did it appear here